root_VS2012/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Framework/Js/common.js

```diff
@@ -0,0 +1,600 @@
+// プログレス ダイアログ表示を仕掛ける。
+// ---------------------------------------------------------------
+// 引数    －
+// 戻り値  －
+// ---------------------------------------------------------------
+function Fx_SetProgressDialog() {
+    if (IsDownload) {
+        // ダウンロードの場合
+
+        // フラグを戻す
+        IsDownload = false;
+    }
+    else {
+        // ダウンロードでない場合
+
+        // ダイアログ表示（２秒後）
+        ProgressDialogTimer = setTimeout("Fx_DisplayProgressDialog()", 2000);
+    }
+}
+
+//**********************************************************************************
+
+// ------------------------------------------------------------
+//  プログレス ダイアログの表示
+// ------------------------------------------------------------
+
+// Ajax：プログレス ダイアログ（div）
+var AjaxProgressDialog;
+
+// Ajax：プログレス ダイアログの表示タイマ
+var ProgressDialogTimer;
+
+// ---------------------------------------------------------------
+// プログレス ダイアログの初期化
+// ---------------------------------------------------------------
+// 引数    －
+// 戻り値  －
+// ★★★  ダイアログのデザインを変える場合は、ここを直接編集
+// ---------------------------------------------------------------
+function Fx_InitProgressDialog() {
+
+    // divを生成   
+    var _div = document.createElement("div");
+    _div.id = "AjaxProgressDialog";
+
+    // 幅を指定
+    _div.style.width = AjaxProgressDialog_Width + "px";
+    _div.style.height = AjaxProgressDialog_Height + "px";
+
+    // スタイルを指定
+    _div.style.top = "0px";
+    _div.style.left = "0px";
+
+    _div.style.paddingTop = "10px";
+    _div.style.paddingLeft = "10px";
+    _div.style.paddingRight = "10px";
+
+    _div.style.textAlign = "center";
+    _div.style.overflow = "auto";
+
+    _div.style.position = "absolute";
+
+    // 1000なら最前面だろうという仕様（ToMost相当が無い）
+    _div.style.zIndex = "1001"; // Maskより前面に出す。
+
+    // 内容を指定
+    _div.innerHTML = "処理中です。しばらくお待ち下さい・・・<hr />";
+    _div.style.backgroundColor = "lightcyan";
+
+    // imgを生成
+    var _img = document.createElement("img");
+
+    _img.src = "/MVC_sample/Framework/Img/loading.gif";
+    _img.style.width = "50px";
+    _img.style.height = "50px";
+    _img.alt = "処理中画像";
+
+    // divにimgを追加
+    _div.appendChild(_img);
+
+    // div → プログレス ダイアログ
+    AjaxProgressDialog = _div
+}
+
+// ---------------------------------------------------------------
+// プログレス ダイアログ表示
+// ---------------------------------------------------------------
+// 引数    －
+// 戻り値  －
+// ---------------------------------------------------------------
+
+function Fx_DisplayProgressDialog() {
+    // はじめにタイマをクリアする。
+    clearTimeout(ProgressDialogTimer);
+
+    try {
+        // 表示位置の計算
+        AjaxProgressDialog.style.top = (Fx_getContentsHeight() / 2) //(Fx_getBrowserHeight() / 2)
+            - (AjaxProgressDialog_Height / 2) + "px";
+        AjaxProgressDialog.style.left = (Fx_getBrowserWidth() / 2)
+            - (AjaxProgressDialog_Width / 2) + "px";
+
+        // プログレス ダイアログを表示する。
+        document.body.appendChild(AjaxMask);
```

Comment:
@daisukenishino 
>Does this pullrequest contains all of the requirements of #129?

Yes, because of merge conflicts, We did all the changes in this pull request.